### DISCUSSION TITLE
Github education got rejected

### DISCUSSION THREAD

**Discussion by xyzbtw** [community/community#121019] [GitHub Education]
### Hi everyone,

I'm currently studying in an online school that is located in the U.S. 
I provided my transcript with my name in it, but it got denied because I live in another country that doesn't have the campus or school building near me (for obvious reasons because I'm in another country).
Is there some way I could still get the github education benefits while studying online in another country?

**Comment by gurjeetsinghvirdee** [comment]
Hey @xyzbtw , 
show them your college ID card. Download it from the online school. Mine has the year on it too. 
I've used mine for verification, and it works just fine.

**Comment by samus-aran** [comment]
Hi @xyzbtw 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. 

I suggest look at acquiring evidence that is more preferred (as @gurjeetsinghvirdee suggested, a Student ID). Another suggestion is to see if your school can provide evidence that you are a remote learner.

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
